schemas/com.snowplowanalytics.snowplow.badrows/loader_runtime_error/jsonschema/1-0-1

```diff
@@ -0,0 +1,524 @@
+            "type": ["number", "null"]
+          },
+          "base_currency": {
+            "type": ["string", "null"],
+            "maxLength": 3
+          },
+          "geo_timezone": {
+            "type": ["string", "null"],
+            "maxLength": 64
+          },
+          "mkt_clickid": {
+            "type": ["string", "null"],
+            "maxLength": 128
+          },
+          "mkt_network": {
+            "type": ["string", "null"],
+            "maxLength": 64
+          },
+          "etl_tags": {
+            "type": ["string", "null"],
+            "maxLength": 500
+          },
+          "dvce_sent_tstamp": {
+            "type": ["string", "null"],
+            "format": "date-time"
+          },
+          "refr_domain_userid": {
+            "type": ["string", "null"],
+            "maxLength": 128
+          },
+          "refr_dvce_tstamp": {
+            "type": ["string", "null"],
+            "format": "date-time"
+          },
+          "domain_sessionid": {
+            "type": ["string", "null"],
+            "maxLength": 128
+          },
+          "derived_tstamp": {
+            "type": ["string", "null"],
+            "format": "date-time"
+          },
+          "event_vendor": {
+            "type": ["string", "null"],
+            "maxLength": 1000
+          },
+          "event_name": {
+            "type": ["string", "null"],
+            "maxLength": 1000
+          },
+          "event_format": {
+            "type": ["string", "null"],
+            "maxLength": 128
+          },
+          "event_version": {
+            "type": ["string", "null"],
+            "maxLength": 128
+          },
+          "event_fingerprint": {
+            "type": ["string", "null"],
+            "maxLength": 128
+          },
+          "true_tstamp": {
+            "type": ["string", "null"],
+            "format": "date-time"
+          },
+          "unstruct_event": { "type": ["object", "null"] },
+          "contexts": { "type": ["object", "null" ]},
+          "derived_contexts": { "type": ["object", "null" ]}
+        },
+        "additionalProperties": false
+      },
+      "failure": {
+        "type": "string",
+        "description": "Error message, either human-readable or stacktrace"
+      },
+      "processor": {
+        "type": "object",
+        "description": "Information about the piece of software responsible for the creation of enrichment failures",
+        "properties": {
+          "artifact": {
+            "type": "string",
+            "description": "Artifact responsible for the creation of enrichment failures",
+            "maxLength": 512
+          },
+          "version": {
+            "type": "string",
+            "description": "Version of the artifact responsible for the creation of enrichment failures",
+            "pattern": "^(\\d+\\.\\d+\\.\\d+.*)$",
+            "maxLength": 32
+          }
+        },
+        "required": [ "artifact", "version" ],
```

Comment:
Why not `processor` ? It's just a nice have?

Reply:
To make it backward-compatible with 1-0-0. I released RDB Loader with this bad row that didn't have a `processor`, so we need to remain compatible with it.